Sources/YCoreUI/Protocols/SystemImage.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SystemImage.swift
+//  YCoreUI
+//
+//  Created by Mark Pospesel on 3/8/23.
+//  Copyright © 2023 Y Media Labs. All rights reserved.
+//
+
+import UIKit
+
+/// Any string corresponding to a system image (SF Symbols).
+///
+/// All properties and functions have default implementations. At a minimum just have your string-based enum conform
+///  to `SystemImage`.  The raw value of the enum should match a sytem image name (e.g. `checkmark.seal`).
+public protocol SystemImage: RawRepresentable where RawValue == String {
+    /// Fallback image to use in case a system image cannot be loaded.
+    /// (default is a 16 x 16 square filled with `.systemPink`)
+    static var fallbackImage: UIImage { get }
+
+    /// A system image for this name value.
+    ///
+    /// Default implementation calls `loadImage` and nil-coalesces to `fallbackImage`.
+    var image: UIImage { get }
+
+    /// Loads the named system image.
+    /// - Returns: The named system image or else `nil` if the system image cannot be loaded.
+    func loadImage() -> UIImage?
+}
+
+extension SystemImage {
+    /// Fallback image to use in case a system image cannot be loaded.
+    /// (default is a 16 x 16 square filled with `.systemPink`)
+    public static var fallbackImage: UIImage {
```

Comment:
They're both intended to be public.
`fallbackImage` so that it can be overridden with a custom image (if desired)
`loadImage()` so that it can also be overridden for a different implementation. Also unit tests should be using `loadImage` and marking it `internal` would require adopters to use `testable import` to access it.